coords:
  src_pose: [0.3, 0.3, 0.2, 0.0, 90.0, 0.0]
  goal_pose: [0.3, 0.2, 0.4, 0.0, 90.0, 0.0]